src/components/Card/__tests__/Card.test.tsx

```diff
@@ -12,28 +12,37 @@ jest.mock('react-router-dom', () => ({
 
 describe('Card', () => {
     it('should render card with single column', () => {
-        var columns = [{key: 'columnKey', value: 'columnValue'}];
-        render(<Card columns={columns} />);
+        const id = 'test-id';
+        const columns = [{key: 'columnKey', value: 'columnValue'}];
+        render(<Card id={id} columns={columns} />);
+
+        expect(screen.queryAllByTestId("card-column").length).toBe(columns.length);
 
         expect(screen.getByText('columnKey')).toBeInTheDocument();
         expect(screen.getByText('columnValue')).toBeInTheDocument();
     });
 
     it('should render card with multiple columns', () => {
+        const id = 'test-id';
         var columns = [
             {key: 'columnKey1', value: 'columnValue1'},
             {key: 'columnKey2', value: 'columnValue2'},
             {key: 'columnKey3', value: 'columnValue3'},
             {key: 'columnKey4', value: ''},
         ];
-        render(<Card columns={columns} />);
+        render(<Card id={id} columns={columns} />);
+
+        expect(screen.queryAllByTestId("card-column").length).toBe(columns.length);
```

Comment:
Added test to check if the number of columns in the created card is the same as the length of the columns passed to the component